### PR TITLE
Update build.yml

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
> patch fix go. Workflow step action not use custom versions. Usually, major versions are bundled with the latest version. Source:https://github.com/actions/toolkit/blob/main/docs/action-versioning.md

Hi 

In your answer, if you check all the items on GitHub, they are the latest stable versions

**Comment by ghost** [comment]
> > > patch fix go. Workflow step action not use custom versions. Usually, major versions are bundled with the latest version. Source:https://github.com/actions/toolkit/blob/main/docs/action-versioning.md
> > 
> > 
> > Hi
> > In your answer, if you check all the items on GitHub, they are the latest stable versions
> 
> No, it's not necessary , actually bind the latest version to a major version. There is no need update, and large versions are updated annually.If you don't understand, please go see github tag.

Thank you for your reply dear friend
We are waiting for more PR from you

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -15,27 +15,33 @@ jobs:
...
       with:
-        go-version: '1.22.4'
+        go-version: '1.23.2'
         cache: false
 
+    - name: Patch Go use 600296
+      #https://go-review.googlesource.com/c/go/+/600296
+      run: |
+        cd "$(go env GOROOT)"
+        curl "https://go-review.googlesource.com/changes/go~600296/revisions/5/patch" | base64 -d | patch --verbose -p 1
```
Comment (UjuiUjuMandan) [inline .github/workflows/build.yml:36]: Why is the patch necessary?

Comment (2dust) [reply .github/workflows/build.yml:36]: https://github.com/golang/go/issues/68285

Comment (UjuiUjuMandan) [reply .github/workflows/build.yml:36]: I've removed the patch but it still works.

- https://github.com/UjuiUjuMandan/v2rayNG/actions/runs/12453055041

**Pull request merged**